db/migrate/20150924230240_create_hydra_shibboleth_user.rb

```diff
@@ -0,0 +1,20 @@
+class CreateHydraShibbolethUser < ActiveRecord::Migration
+  def change
+    create_table :hydra_shibboleth_users do |t|
+      t.string :email,              :null => true, :default => ""
```

Comment:
I think maybe it doesn't matter if give empty strings as default or not, there would be slight difference when check those fields using nil? vs. empty?